server/src/remote.rs

```diff
@@ -457,13 +457,19 @@ impl FlightService for FlightServiceHandler {
 
     /// Perform a specific action based on the type of the action in `request`.
     /// Currently only the action `CommandStatementUpdate` is supported which
-    /// executes a SQL query containing a single command that does not return a
-    /// result. These commands can be `CREATE TABLE table_name(name type*)`
-    /// which creates a normal table in the catalog, and `CREATE MODEL TABLE
-    /// table_name(name_one TIMESTAMP, name_two FIELD*, name_three,
-    /// FIELD(error_bound)*, name_four TAG*)` which creates a model table in the
-    /// catalog. A model table is a specialized table for efficiently storing
-    /// multivariate time series with tags within a per field error bound.
+    /// executes a SQL query containing a command that does not return a result.
+    /// These commands can be `CREATE TABLE table_name(...` which creates a
+    /// normal table, and `CREATE MODEL TABLE table_name(...` which creates a
+    /// model table. A model table is a specialized table for efficiently
+    /// storing multivariate time series with tags within a per field error
+    /// bound. Thus, model tables can only contain a single timestamp column,
+    /// field columns, and tag columns. If no error bound is defined for a field
+    /// column it defaults to an error bound of 0% (lossless compression).
+    ///
+    /// Examples of CREATE TABLE and CREATE MODEL TABLE commands:
+    /// * CREATE TABLE company(id INTEGER, name TEXT)
+    /// * CREATE MODEL TABLE wind_turbines(timestamp TIMESTAMP,
+    ///   field_lossless FIELD, field_lossy FIELD(5),  tag_one TAG, tag_two TAG)
```

Comment:
Very minor but it seems that there is two spaces between `,` and `tag_one`.

Reply:
Fixed.